buildscripts/kokoro/windows.bat

```diff
@@ -0,0 +1,40 @@
+@rem ##########################################################################
+@rem
+@rem Script to set up Kokoro worker and run Windows tests
+@rem
+@rem ##########################################################################
+
+@rem Enter repo root
+cd /d %~dp0\..\..
+
+set WORKSPACE=T:\src\github\grpc-java
+set ESCWORKSPACE=%WORKSPACE:\=\\%
+
+mkdir grpc-java-helper
+
+@rem Install 7za
+wget http://www.7-zip.org/a/7za920.zip
+unzip 7za920.zip
```

Comment:
Yeah, although it is a nice feature that `make_dependencies.bat` can be run manually. But depending on unzip vs 7za doesn't really make it any harder to run manually.